pkg/query/filter/error.go

```diff
@@ -25,7 +23,7 @@ func NewValidationError(format string, value ...any) *ValidationError {
 }
 
 func NewValidationErrorWithStack(format string, value ...any) error {
-	return errors.WithStack(NewValidationError(format, value...))
+	return NewValidationError(format, value...)
 }
 
```

Comment:
function can be removed

Reply:
Fixed in 09ed94b786c985ba136235213403fa2063edf3af